EIPS/eip-1559.md

```diff
@@ -48,15 +48,15 @@ The transition to this gas price system will occur in two phases, in the first p
 * `FINAL_FORK_BLKNUM`: `INITIAL_FORK_BLKNUM + MIGRATION_DURATION_IN_BLOCKS`
 * `BLOCK_GAS_TARGET`: The gas limit field set in the block header, controlled by miner voting.
 * `EIP1559_INITIAL_GAS_TARGET`: `BLOCK_GAS_TARGET / 2`
-* `LEGACY_INITIAL_GAS_LIMIT`: `BLOCK_GAS_TARGET - EIP1559_GAS_TARGET`
+* `LEGACY_INITIAL_GAS_LIMIT`: `BLOCK_GAS_TARGET / 2`
 * `EIP1559_GAS_TARGET`:
   ```
   if CURRENT_BLKNUM >= FINAL_FORK_BLOKNUM then
     BLOCK_GAS_TARGET
   elif CURRNT_BLKNUM < INITIAL_FORK_BLKNUM then
     0
   else
-    EIP1559_INITIAL_GAS_TARGET + LEGACY_INITIAL_GAS_LIMIT * (CURRENT_BLKNUM - INITIAL_FORK_BLKNUM) / MIGRATION_DURATION_IN_BLOCKS
+    EIP1559_INITIAL_GAS_TARGET + (BLOCK_GAS_TARGET / 2) * (CURRENT_BLKNUM - INITIAL_FORK_BLKNUM) / MIGRATION_DURATION_IN_BLOCKS
```

Comment:
Same comment as above, by spreading out the `/2` rather than using variables, if we decide we want to change the initial distribution we have to update more parts of the EIP (more likely to miss/forget something and introduce a bug in the spec).  This change isn't incorrect, it just has *potential* to make the EIP a tad harder to maintain.